### PR TITLE
Update Frame Tutorials for current state of OnchainKit

### PR DESCRIPTION
**Pull request by briandoyle81CB**
**What changed? Why?**
Update Frame Tutorials for current state of OnchainKit.  

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: apps/base-docs/tutorials/docs/3_farcaster-frames-deploy-to-vercel.md [hunk 1]
```diff
@@ -56,13 +50,39 @@ You must have a [Farcaster] account with a connected wallet. Check out the [Base
 
 ## Setup and Testing the Template
 
-Start by creating a fork of the [a-frame-in-100-lines].
+Start by creating a new repo using [a-frame-in-100-lines] as a template.
```
Comment (hughescoin) [inline apps/base-docs/tutorials/docs/3_farcaster-frames-deploy-to-vercel.md:53]: Does this automatically link to the github page or are you missing the url?

Comment (hughescoin) [reply apps/base-docs/tutorials/docs/3_farcaster-frames-deploy-to-vercel.md:53]: Ignore that. Didn't have the full view

**Review by hughescoin — DISMISSED** [review]

**Comment by cb-heimdall** [comment]
Review Error for hughescoin @ 2024-07-12 16:34:01 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

File: apps/base-docs/tutorials/docs/3_farcaster-frames-gating-and-redirects.md [hunk 2]
```diff
@@ -186,12 +144,12 @@ You can now add outbound links to buttons. To do this with [OnchainKit], simply
 
 ### Redirect Button
 
-The third button contains a `Redirect to pictures ↗`. The way it works is a little tricky. Start with the `buttons` defined in the original frame in `page.tsx`. The third button is a `post_redirect`:
+You can also do a redirect with a button. The example has one on the second frame - `Dog pictures ↗`. The way it works is a little tricky. Start with the `buttons` defined in the response frame in `app/frame/route.tsx`. The third button is a `post_redirect`:
```
Comment (hughescoin) [inline apps/base-docs/tutorials/docs/3_farcaster-frames-gating-and-redirects.md:148]: I wasn't sure if you are telling me to add a redirect button or suggesting. Can you clarify?

Comment (hughescoin) [reply apps/base-docs/tutorials/docs/3_farcaster-frames-gating-and-redirects.md:148]: Should the user have 4 or 3 buttons?

```javascript
buttons: [
    {
      label: 'Story time',
    },
    {
      action: 'tx',
      label: 'Send Base Sepolia',
      target: `${NEXT_PUBLIC_URL}/api/tx`,
      postUrl: `${NEXT_PUBLIC_URL}/api/tx-success`,
    },
    {
      action: 'link',
      label: 'Link to Google',
      target: 'https://www.google.com',
    },
    {
      action: 'post_redirect',
      label: 'Dog pictures',
    },
  ],
```

File: apps/base-docs/tutorials/docs/3_farcaster-frames-gating-and-redirects.md [hunk 3]
```diff
@@ -244,42 +202,46 @@ if (message?.button === 3 && message.liked) {
...
+        src: `${NEXT_PUBLIC_URL}/park-1.png`,
       },
       postUrl: `${NEXT_PUBLIC_URL}/api/frame`,
+      state: {
+        page: state?.page + 1,
+        time: new Date().toISOString(),
+      },
     }),
   );
 }
```
Comment (hughescoin) [inline apps/base-docs/tutorials/docs/3_farcaster-frames-gating-and-redirects.md:227]: I needed to add a default return statement. typescript linter gave me an error

I added:
```
  return new NextResponse('Unhandled request', { status: 400 });
  ```

Comment (briandoyle81CB) [reply apps/base-docs/tutorials/docs/3_farcaster-frames-gating-and-redirects.md:229]: You're supposed to leave the original return of a frame at the bottom.  Added a note.

**Review by hughescoin — APPROVED** [review]

**Pull request merged**